modules/cnvkit/CHANGELOG.md

```diff
@@ -0,0 +1,32 @@
+# Changelog
+
+All notable changes to the `cnvkit` module will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [1.0] - 2022-01-22
+
+This release was authored by Jasper Wong.
+
+Cnvkit uses both on-target and off-target bait regions to determine CNV.
+
+Everything is bundled under one function "batch", which builds a normal reference from a panel of normals and calls CN for each tumour.
```

Comment:
Thanks Jasper! Can you please update CHANGELOG to reflect the recent overhaul and I'll merge it to master 🙏

Reply:
Okay done